share/completions/iwctl.fish

```diff
@@ -6,7 +6,7 @@ set dpp '(iwctl dpp list | tail -n -2 | awk \'{print $1}\')'
 set knownnetworks '(iwctl known-networks list | tail -n -2 | awk \'{print $1}\')'
 set stations '(iwctl station list | tail -n -2 | awk \'{print $1}\')'
 set wsc '(iwctl wsc list | tail -n -2 | awk \'{print $1}\')'
```

Comment:
These others also need to have the `tail` adjusted. `tail -n -2` will give you the last two lines, which only works if you have one entry. We want all of them, we just want to skip the header. That's `tail -n +5`, to start at the fifth line of output.

I took a quick glance, as far as I can see they don't mark a "current" or "active" entry so you can probably skip the final `replace`.

You might have to strip the colors tho.